src/content/reference/react/Fragment.md

```diff
@@ -4,7 +4,7 @@ title: <Fragment> (<>...</>)
 
 <Intro>
 
-`<Fragment>`, often used via `<>...</>` syntax, lets you group elements without a wrapper node.
+`<Fragment>`, atau sering digunakan melalui sintaksis `<>...</>`, memungkinkan Anda mengelompokkan elemen tanpa pembungkus *node*.
```

Comment:
No need for `atau` because there's no `or` from the source. Also include the original word.

```suggestion
`<Fragment>`, sering digunakan melalui sintaksis `<>...</>`, memungkinkan Anda mengelompokkan elemen tanpa pembungkus (*wrapper*) *node*.
```

Reply:
Hi @mhaidarhanif, I already change the word to your suggestion. Please help me to review it again.

Reply:
@AudiWu Got it